pkg/orchestrator/assetscanestimationwatcher/watcher.go

```diff
@@ -237,9 +284,20 @@ func (w *Watcher) reconcilePending(ctx context.Context, assetScanEstimation *mod
 
 	assetScanEstimation.State.LastTransitionTime = utils.PointerTo(time.Now())
 
+	// Set default ttl if not set.
+	if assetScanEstimation.TTLSecondsAfterFinished == nil {
+		assetScanEstimation.TTLSecondsAfterFinished = utils.PointerTo(DefaultAssetScanEstimationTTLSeconds)
+	}
+	endTime := time.Now()
```

Comment:
can you explain why the `endTime` and `startTime` is now?

Reply:
It is basically because the action of Estimate is really fast, but you are right, I will move the startTime to before calling Estimate